styles/wikipedia/catppuccin.user.css

```diff
@@ -256,10 +256,266 @@
 
     .mw-highlight pre {
       background: @surface0 !important;
-      border-color: @surface2 !important;
+      border-width: 0;
```

Comment:
Oh wait sorry, just saw this. Is this avoidable?

Reply:
Sure, can be undone if you wish.

Reply:
```css
border-color: @surface0 !important;
```

Reply:
But, it makes the border look like this:

![](https://media.discordapp.net/attachments/1085327542447259740/1213822937015001128/image.png?ex=65f6dfb3&is=65e46ab3&hm=858f27b86a445bafdac9bdd0d0b267dc71b74ea1c0d9d56701ee3909b0b41e96&=&format=webp&quality=lossless&width=242&height=194)

Reply:
Alright, that makes sense.